Tests/Docker/BillingClientTest.php

```diff
@@ -110,7 +110,7 @@ public function testClientRequestResponse()
         $stack->push($history);
         $client = $this->getClient(['handler' => $stack]);
         $credits = $client->getRemainingCredits();
-        self::assertEquals(123.43434343434, $credits);
+        self::assertEqualsWithDelta(123.43434343434, $credits, 0.0001);
```

Comment:
Je to float a občas to napíše, že `123.43434343434` is not equal to `123.43434343434`